### DISCUSSION TITLE
MacBook Pro 13,2 Touch-ID

### DISCUSSION THREAD

**Discussion by Samir-eus** [community/community#37550] [Other Feature Feedback, Questions, & Ideas] [answered]
I install macOS Ventura on unsupported MacBook Pro 13,2 by using you great (OpenCore Legacy Patcher 5.0) and everything fine else Touch-ID doesn’t appear in system setting

**Comment by antonblr** [comment, marked as answer]
Hi @Samir-eus - this community is for GitHub related topics. For MacOS  related questions / issues / topics (Ventura beta etc) you'd get better chances at https://developer.apple.com/forums/
